forbidigo/forbidigo_test.go

```diff
@@ -146,12 +159,14 @@ func expectIssues(t *testing.T, linter *Linter, contents string, issues ...strin
 }
 
 func parseFile(t *testing.T, linter *Linter, fileName, contents string) []Issue {
-	fset := token.NewFileSet()
-	expr, err := parser.ParseFile(fset, fileName, contents, parser.ParseComments)
+	pass := &analysis.Pass{
```

Comment:
Since we aren't taking an analysis Pass as an argument any more, could we avoid adding this as a dependency here?

Reply:
Yes, that's left from the initial implementation. I've removed the `Pass` instance ([force-pushed](https://github.com/ashanbrown/forbidigo/compare/d080f74ceda4564c08558cf1a93089c7e2df776e..8dde8a800a83a715777123951e08e3c037431dd1) here and also in the other PR).